go.mod

```diff
@@ -24,3 +24,5 @@ require (
 	k8s.io/client-go v0.17.17
 	sigs.k8s.io/controller-runtime v0.5.0
 )
+
+replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
```

Comment:
Let's pull this in as an indirect dependency. Replaces can have issues when upgrading deps later on, so I'd prefer to avoid them.

Running this command pulled in the new version to `go.mod` for me:
```bash
go get github.com/gogo/protobuf@latest
```

Reply:
@JohnStarich , fixed per comment,  Also, the softlayer-go package is updated along with the `go mod tidy` cmd

Reply:
@JohnStarich , can you review again?